forum/forum/utils.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+from django.template.loader import render_to_string
+
+
+def build_email_message(
+    email_template_path: str,
+    context: dict[str, Any] = None
+) -> str:
```

Comment:
Adding a None check for context to avoid possible issues when rendering the template.
    if context is None:
        context = {}

Reply:
There is a signature of the function.
```Python
def render_to_string(
    template_name: list[str] | str,
    context: dict[str, Any] | None = ...,
    request: HttpRequest | None = ...,
    using: str | None = ...
) -> str
```

And there is the function which the task calls. It's allowed to provide `context` variable as None value to this function. So it's possible to omit checking for None-type.
```Python
def render_to_string(template_name, context=None, request=None, using=None):
    """
    Load a template and render it with a context. Return a string.

    template_name may be a string or a list of strings.
    """
    if isinstance(template_name, (list, tuple)):
        template = select_template(template_name, using=using)
    else:
        template = get_template(template_name, using=using)
    return template.render(context, request)

```